kv/txn_coord_sender.go

```diff
@@ -309,6 +311,10 @@ func (tc *TxnCoordSender) Send(_ context.Context, call proto.Call) {
 	header := call.Args.Header()
 	tc.maybeBeginTxn(header)
 
+	header.CmdID = header.GetOrCreateCmdID(tc.clock.PhysicalNow())
+	tc.tracer.Begin(header, "coordsender_begin")
```

Comment:
The suffixes "_begin" and "_done" should be added by Begin and Finalize to ensure they're consistent everywhere.